tools/kompat/go.mod

```diff
@@ -1,19 +1,20 @@
 module github.com/aws/karpenter-provider-aws/tools/kompat
 
-go 1.22
+go 1.24
 
 require (
-	github.com/Masterminds/semver/v3 v3.2.1
+	github.com/Masterminds/semver/v3 v3.4.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/olekukonko/tablewriter v0.0.5
```

Comment:
bumping `github.com/olekukonko/tablewriter` would need to do refactoring, I think it would be better kept in another commit.